forge/ee/db/controllers/Subscription.js

```diff
@@ -5,6 +5,10 @@ module.exports = {
         if (existingSub) {
             existingSub.customer = customer
             existingSub.subscription = subscription
+            if (existingSub.status === app.db.models.Subscription.STATUS.TRIAL) {
+                existingSub.trialEndsAt = null
+                existingSub.trialStatus = app.db.models.Subscription.TRIAL_STATUS.ENDED
+            }
```

Comment:
This is handling the 'subscription created' callback from stripe - at which point we clear the trial status of the local subscription object.